apps/src/templates/sectionAssessments/MatchAssessmentsOverviewTable.story.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {UnconnectedMatchAssessmentsOverviewTable} from './MatchAssessmentsOverviewTable';
+import {
+  matchQuestionWith2Pairs,
+  matchQuestionWith4Pairs
+} from './assessmentsTestHelpers';
```

Comment:
this is a good example a test fixture, which we are trying to move toward using more often. thanks for adding this!